### PR TITLE
Update the ACME github repository URL.

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

**Comment by kuba** [comment]
LGTM, however there are other things to update there (such us latest supported spec revision) which could be done in the same PR.

**Comment by Natim** [comment]
> latest supported spec revision

@kuba I was looking for that but I have no idea of which version is currently supported and I couldn't find the previous commit in the new github.

**Comment by kuba** [comment]
We are (partially) implementing version `draft-ietf-acme-acme-01` (available at https://github.com/ietf-wg-acme/acme/tree/draft-ietf-acme-acme-01)

**Comment by Natim** [comment]
Should I explicit that in the file then? and chanve v02 by https://github.com/ietf-wg-acme/acme/tree/draft-ietf-acme-acme-01 ?

**Comment by kuba** [comment]
Please change inline text `v02` to `draft-ietf-acme-01` and swap the links. Thanks

**Comment by Natim** [comment]
Updated.

File: acme/acme/__init__.py [hunk 1]
```diff
@@ -3,10 +3,10 @@
 This module is an implementation of the `ACME protocol`_. Latest
 supported version: `v02`_.
 
-.. _`ACME protocol`: https://github.com/letsencrypt/acme-spec
+.. _`ACME protocol`: https://github.com/ietf-wg-acme/acme/
 
-.. _`v02`:
-  https://github.com/letsencrypt/acme-spec/commit/d328fea2d507deb9822793c512830d827a4150c4
+.. _`draft-ietf-acme-01`:
```
Comment (kuba) [inline acme/acme/__init__.py]: you have to also change `v02` to `draft-ietf-acme-01` throughout the text

Comment (Natim) [inline acme/acme/__init__.py]: I actually search for it and couldn't find any references.
Did you?

Comment (kuba) [inline acme/acme/__init__.py]: yes, e.g. 4 lines above, `cd acme; git grep v02`

Comment (Natim) [inline acme/acme/__init__.py]: Oh dear, you are right. Updated.

**Comment by kuba** [comment]
LGTM, though someone else will have to merge this

**Comment by bmw** [comment]
LGTM as well. Thanks for the PR @Natim.

**Pull request merged**